.github/workflows/test.yaml

```diff
@@ -21,6 +21,15 @@ jobs:
         uses: bigladder/github-actions/setup-python-poetry@main
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Lint with Pylint
+        run: |
+          poetry run pylint atheneum #'${{github.workspace}}
+          poetry run pylint test/*.py
+        continue-on-error: true
+      - name: Static type checking
+        run: |
+          poetry run mypy atheneum #'${{github.workspace}}
```

Comment:
Do we want to run this on our tests, too?

Reply:
I was thinking not, actually. If we're writing tests that are typing-complicated, we have bigger problems. But I guess I don't mind either way.